static/css/thing.css

```diff
@@ -374,12 +374,13 @@
 }
 
 .media-modal {
-  width: calc(100% - 15.2rem);;
+  width: calc(100% - 15.2rem);
```

Comment:
This should be `max-width`, otherwise you will mess up the aspect ratio of videos and images.

Reply:
I haven't actually changed anything here :-) I thought I'd keep that for a separate PR.

Reply:
Ah, fair enough.